clients/drcachesim/simulator/snoop_filter.cpp

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
Please add the copyright and license header comments.

Reply:
Added.